icons/talos.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Talos</title><path d="M9.4 12c0-2.7-1.3-7-3.1-10.8-.7.2-1.2.6-1.8 1 1.8 3.5 3 7.5 3 9.8a26 26 0 0 1-3.1 9.9l1.7 1c2-4.1 3.3-8.4 3.3-10.9m7.2 0c0-2.3 1-6.3 3-9.9a14 14 0 0 0-2-1c-1.8 4-3 8.2-3 10.9 0 2.5 1.3 6.8 3.3 10.8a13 13 0 0 0 1.6-1c-1.7-3.6-2.9-7.6-2.9-9.8m6 6.5.4.3 1-1.6c-2.1-2-3.9-4-3.9-5.2 0-1.3 1.8-3.3 4-5.2A12 12 0 0 0 23 5l-.4.3c-3 2.8-4.5 4.9-4.5 6.6s1.6 3.8 4.5 6.5M1.4 5.4l-.3-.3a12 12 0 0 0-1 1.7C3.5 10 3.8 11.5 3.8 12c0 1.3-1.8 3.3-3.9 5.2l1 1.6.4-.3C4.4 15.8 6 13.7 6 12S4.5 8.2 1.5 5.4M12 24h1V0h-2v24z"/></svg>
```

Comment:
Thank you @buroa for your contribution. I think you stretched your icon to fit the 24x24 box. In fact, only the longer side need to have 24px in size. Right now, comparing your version (in black) with the original gives this:
![image](https://github.com/simple-icons/simple-icons/assets/12817388/c1172c27-9e1c-4235-8648-f99603817877)

Can you try this again?

Reply:
@service-paradis Can you check it again?